doc/getting_started.rst

```diff
@@ -10,7 +10,7 @@ Run ``pip install metric-learn`` to download and install from PyPI.
 Alternately, download the source repository and run:
 
 -  ``python setup.py install`` for default installation.
--  ``python setup.py test`` to run all tests.
+-  ``pytest test`` to run all tests.
```

Comment:
Indeed it seems that `python setup.py test` has been deprecated:
https://setuptools.readthedocs.io/en/latest/setuptools.html#test-build-package-and-run-a-unittest-suite
They now recommend to use `tox` instead, which seems quite heavy

So I am happy with the proposed solution of simply suggesting to run `pytest`